src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1539,11 +1539,15 @@ public static void writeTypedPackFormats(JsonObject json, PackMetadataSection se
             int format = section.getPackFormat(packType);
             if (format != packFormat)
             {
-                json.addProperty("forge:" + packType.bridgeType.name().toLowerCase(Locale.ROOT) + "_pack_format", format);
+                json.addProperty(makePackFormatKey(packType), format);
             }
         }
     }
 
+    private static String makePackFormatKey(PackType packType) {
```

Comment:
Formatting: braces on newlines.
```suggestion
    private static String makePackFormatKey(PackType packType)
    {
```